api/destination_credential.go

```diff
@@ -0,0 +1,67 @@
+package api
+
+import (
+	"github.com/infrahq/infra/uid"
+)
+
+type CreateDestinationCredential struct {
+	Destination string `form:"destination"`
+}
+
+type ListDestinationCredential struct {
+	Destination     string `form:"destination"`
+	LastUpdateIndex int64  `form:"lastUpdateIndex"`
+}
+
+type ListDestinationCredentialResponse struct {
+	Items          []DestinationCredential `json:"items"`
+	MaxUpdateIndex int64                   `json:"maxUpdateIndex"`
+}
+
+// DestinationCredential is the database DestinationCredential object, not a request/response struct
+type DestinationCredential struct {
+	////////////////////
+	// request fields //
+	////////////////////
+	ID             uid.ID `json:"id"`
+	OrganizationID uid.ID `json:"organizationID"`
+
+	UserID      uid.ID `json:"userID"`
+	Destination string `json:"destination"`
```

Comment:
Should this be a destination ID also?

Reply:
Maybe, but I think that makes it harder for the user to use the `infra credentials add dest` command.